### PR TITLE
Python Code Update

### PR DESCRIPTION
**Pull request by andrewmzhang**
Plotly UI and Jupyter Notebook example updates

### CODE REVIEW

**Comment by andrewmzhang** [comment]
I can fix the merge conflict easily, but I recommend we merge https://github.com/jcarreira/cirrus/pull/50 first, since there are overlapping changes.

File: python/frontend/cirrus/.gitignore [hunk 1]
```diff
@@ -68,3 +68,171 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
```
Comment (jcarreira) [inline python/frontend/cirrus/.gitignore]: Can you clean this? Most of these things we don't use

File: python/frontend/cirrus/cirrus/utils.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (jcarreira) [inline python/frontend/cirrus/cirrus/utils.py]: Remove this spaces.

File: python/frontend/cirrus/cirrus/utils.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+import random
+
+def get_random_color():
+    r = lambda: random.randint(0,255)
+    return 'rgb(%d, %d, %d)' % (r(),r(),r())
+
+
+def command_dict_to_file(command_dict):
```
Comment (jcarreira) [inline python/frontend/cirrus/cirrus/utils.py]: What does this do? Need comments

File: src/Makefile.am [hunk 4]
```diff
@@ -10,7 +10,7 @@ bin_PROGRAMS = parameter_server ps_test csv_to_libsvm $(TOP_DIR)/tests/test_trav
 
 LIBS         =  -laws-cpp-sdk-s3 -laws-cpp-sdk-core \
 		-lcurl -lssl -lcrypto -lz -ldl -lkrb5 -lk5crypto \
-		-lall -lkeyutils -lresolv -lgflags
```
Comment (jcarreira) [inline src/Makefile.am]: I think you are committing the Ubuntu Makefile here.

File: python/frontend/cirrus/setup.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    from distutils.core import setup, find_packages
+
+
+setup(
+    name='cirrus',
+    version='0.1dev',
+    description='Python library for Cirrus',
+    author="João Carreira",
+    author_email='joao [at] berkeley [dot] edu',
+    # NOTE: Email on your berkeley page was like this, not sure if you're afraid of bots parsing your github
```
Comment (jcarreira) [inline python/frontend/cirrus/setup.py]: Should remove this. joao@berkeley.edu is fine.

File: python/frontend/cirrus/cirrus/utils.py (hunk 2 above)
Comment (andrewmzhang) [reply python/frontend/cirrus/cirrus/utils.py]: I removed the extra spaces, though in python proper style says there should be 2 new lines between functions

**Pull request merged**